Jenkinsfile

```diff
@@ -175,6 +224,11 @@ pipeline {
             }
             steps {
                 sh '''
+                    kubectl get namespace | awk '/-scf|-uaa/ {print $1}' | xargs --no-run-if-empty kubectl delete ns
+                    while kubectl get namespace | grep 'scf|uaa'; do
```

Comment:
Your `awk` and `grep` expressions don't match; it's possible that you'll end up never exiting this loop because you never deleted a matching namespace.

Reply:
Added dashes to the grep